www/nginx/src/opnsense/scripts/nginx/read_log.php

```diff
@@ -64,25 +64,23 @@
                 die('{"error": "The server entry has no server name"}');
             }
             $lines = [];
-            foreach (explode(',', $server_names) as $server_name) {
-                $log_file_name = $log_prefix . basename($server_name) . '.' . $mode . $log_suffix;
-                // this entry has no log file, ignore it
-                if (!file_exists($log_file_name)) {
-                    continue;
-                }
-                $logparser = null;
+            $log_file_name = $log_prefix . basename($server_names) . '.' . $mode . $log_suffix;
```

Comment:
basename($server_names) looks strange, but you know your code better than me

Reply:
Ok, the logs for http servers are the server names (comma separated).access.log or .error.log (could have used the uuid as well but this way it is easier to work with when working on cli.

The streams do not have something readable so it is just the uuid. The frontend queries all logs except one by uuid. The exception is the global error log which catches everything that is not related to any server.

Reply:
ok, this can be problematic on hostname rename but in general it sounds ok, thanks for explaining